.github/workflows/publish-example-images.yaml

```
@@ -55,8 +55,7 @@ jobs:
           - component-name: mxnet-auto-tuning
             dockerfile: examples/mxnet/tune/Dockerfile
             context: examples/mxnet/tune
-# TODO (tenzen-y): Fix the below broken Dockerfiles
-#          - component-name: pytorch-dist-mnist-mpi
-#            dockerfile: examples/pytorch/mnist/Dockerfile-mpi
-#          - component-name: pytorch-dist-mnist
-#            dockerfile: examples/pytorch/mnist/Dockerfile
+          - component-name: pytorch-dist-mnist
+            dockerfile: examples/pytorch/mnist/Dockerfile
+          - component-name: pytorch-dist-mnist-mpi
+            dockerfile: examples/pytorch/mnist/Dockerfile-mpi
```

```suggestion
          - component-name: pytorch-dist-mnist
            dockerfile: examples/pytorch/mnist/Dockerfile
            context: examples/pytorch/mnist            
          - component-name: pytorch-dist-mnist-mpi
            dockerfile: examples/pytorch/mnist/Dockerfile-mpi
            context: examples/pytorch/mnist
```

Specifying the `context` should resolve CI errors.